src/index.css

```diff
@@ -0,0 +1,187 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/cyptobg.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] When the wireframe/template was provided for this project, one of the requirements was to use the fonts, font size and font weights, margins and spaces accordingly. Kindly stick to this requirement and use `Lato` and `Gill Sans` fonts for this project. 👍